src/components/ImageView/Toolbar/ToolbarComponent.tsx

```diff
@@ -193,8 +193,20 @@ export class ToolbarComponent extends React.Component<ToolbarComponentProps> {
             </Menu>
         );
 
+        const baseFrame = this.props.frame;
+        let numSourcesArray = [];
```

Comment:
Perhaps we don't need to calculate the exact sum of source number. We can apply `Array.prototype.every()` to check if each element is zero.